README.md

```diff
@@ -58,18 +58,21 @@ getting the response with the return value.
 ## Usage example:
 
 ### Server:
+
 ```python
 import uvicorn
 from fastapi import FastAPI
-from fastapi_websocket_rpc import RpcMethodsBase, WebsocketRPCEndpoint
+from fastapi_ws_rpc import RpcMethodsBase, WebsocketRPCEndpoint
+
 
 # Methods to expose to the clients
 class ConcatServer(RpcMethodsBase):
-    async def concat(self, a="", b=""):
-        return a + b
+  async def concat(self, a="", b=""):
```

Comment:
Is this right? It looks like indentation is wrong here, no?

Reply:
2 indent space is fine right?

Reply:
No. Python uses 4 spaces [[ref](https://peps.python.org/pep-0008/#indentation)].

Reply:
Yeah correct, so I added 4, On the change request it seems 2, so I have changed the README.md to 2 indent. Can we follow 4 indent in README.md as well?

Reply:
Yes, everywhere should have 4.